src/store/notifications/notificationSlice.ts

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { OSNotification } from 'react-native-onesignal'
+
+export type NotificationState = {
+  notification?: OSNotification
```

Comment:
I think we should move the notifications in `src/store/account/accountSlice.ts` over to here.

Reply:
Ah ok I can, maybe I call this new one pushNotifcation?

Reply:
that sounds good.